src/clj/rems/db/test_data.clj

```diff
@@ -1129,7 +1164,36 @@
                              :resource-id res-organization-owner
                              :form-id form-organization-owner
                              :organization {:organization/id "organization1"}
-                             :workflow-id (:organization-owner workflows)})))
+                             :workflow-id (:organization-owner workflows)})
+
+    (let [applicant (users :applicant1)
+          handler (users :approver2)
+          reviewer (users :reviewer)
+          catid-1 (create-catalogue-item! {:actor owner
+                                           :title {:en "Test workflow"
+                                                   :fi "Testityövuo"
+                                                   :sv "Test arbetsflöde"}
```

Comment:
Since this is actually using the default workflow perhaps this title should be something else? Like "Default workflow with public and private fields" to be very exact?

Reply:
changed